code/src/components/ThoughtInput.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+export const ThoughtInput = ({ eachThought, onEachThoughtChange, onFormSubmit }) => {
+  return (
+    <form className="thoughtInput" onSubmit={onFormSubmit}>
+      <h1>What&apos;s making you happy right now?</h1>
+      <textarea
+        value={eachThought}
+        onChange={onEachThoughtChange}
+        placeholder="React is making me happy!" />
+      <div className="characterCount">{eachThought.length < 5 || eachThought.length > 140 ? (
+        <p className="redText">{eachThought.length}/140</p>
+      ) : (
+        <p className="normalText">{eachThought.length}/140</p>
+      )}
+      </div>
```

Comment:
Nice with the change depending on the characters, another way of getting the className to change with less code could be:
`<p className={eachThought.length < 5 || eachThought.length > 140 ? 'redText' : 'normalText'}>{eachThought.length} / 140</p>`